pending/contributing.pod

```diff
@@ -0,0 +1,122 @@
+=head1 Contributing to Dancer
+
+The Dancer community is a fun, active, and engaged community, dedicated to 
+the advancement of Dancer and the community as a whole. The core team, in 
+particular, relies on the community for bug reports, feature requests, and
+other information to find out how Dancer is used, what needs fixing, what
+needs improving, and how we can make the framework better for you. This
+guide is intended to show you, at a high level, how to get involved in the 
+community, and things you can do to help further Dancer development and the 
+community. 
+
+Note: This applies primarily to Dancer2, but works for Dancer1 in some 
+regards. Dancer1 is not open for new feature development, but is actively
+maintained in a support-only mode; bug fixes and security patches are 
+incorporated in as timely a manner as possible. If you have a feature 
+idea and are using Dancer1, check the docs to see if it has already 
+been implemented, and if not, get hacking on Dancer2!
+
+=head2 Report a Bug
+
+This is perhaps the easiest way to get involved, and one of the most important
+for the project. If you run into a bug while using Dancer - be it in the 
+core framework, a plugin - B<please> report it! While you can report it in
+Dancer's IRC channel, it is not the most reliable reporting method available
+(that's not to say your report will get ignored there!).
+
+Development of Dancer happens on Github. Your best bet for reporting a bug
+in Dancer is to go to the L<issue tracker on GitHub|https://github.com/PerlDancer/Dancer2/issues>.
+Before submitting a new bug, please be courteous and look to see if there
+is already an open issue for the problem you are observing, and if so, add
+whatever additional information you have to the existing issue.
+
+No matter how well we test Dancer during development, there is no way for us
+to find all of the potential bugs that may arise. The Core Team and the Dancer
+community depends on B<you> to report bugs that you find along the way!
+
+=head2 IRC
+
+A great way to participate and contribute to the Dancer project is to be active
+on the IRC channel. The Dancer Core Team strives to maintain a friendly and civil
+environment on the channel at all times, and users of C<#dancer> would generally
```

Comment:
Instead of the sentence "and users of `#dancer` would...", I would just add "and to all users".